docker-compose.yml

```diff
@@ -8,7 +8,7 @@ x-common-variables: &common-variables
   JWT_SECRET: EXAMPLE_SECRERT # pass around JWT secret so each service can verify user TODO: add to github secrets on deployment
   REFRESH_COOKIE_KEY: EXAMPLE_PEERPREPREFRESHTOKEN
   ACCESS_COOKIE_KEY: EXAMPLE_PEERPREPACCESSTOKEN
```

Comment:
Quick check for cookies and secrets, are they the same for all services? If so we can consider creating a shared secret and shared configmap instead of copying to all configmap perhaps?